src/backend/aspen/database/models/lineages.py

```diff
@@ -106,13 +108,61 @@ class SampleLineage(idbase):  # type: ignore
     lineage_probability = Column(Float, nullable=True)
     raw_lineage_output = Column(JSONB, nullable=True)
 
+    # For Nextclade, we need to track the underlying reference data bundle
+    # that was involved to know how the lineage call was made.
+    # TODO FIXME: Lots of table/row duplication here. _Probably_ better to
+    # normalize it as a single entry somewhere and then reference that.
+    # Evaluate later -- because of how we use this, not hard to change later.
+    # Search this to find all instances: §NextcladeTagDuplicationTODO
+    reference_dataset_name = Column(String, nullable=True)
```

Comment:
sidney said the dataset name is fairly stable and should rarely change.

Reply:
You're right, this should be duplicating the info 99.9% of the time, but it gives us an unambiguous way to have all the call info together. It's related to pulling the info out of the `tag.json` file that comes with the nextclade dataset. In case things ever get muddled, I wanted to consider the `tag.json` the real source of truth.

Alternatively, we could raise an error in the workflow if the `tag.json` name and the pathogens.nextclade_dataset_name were different.

Reply:
Oh, I forgot, the other benefit is that if we ever changed the `pathogens.nextclade_dataset_name` for a given pathogen in the future, that would cause it to auto-trigger a new run on all the impacted samples. That should be a very rare event, but it means that we definitely wouldn't need to go manually trigger the run for those samples. (But that might also be silly, if we change the name, the underlying accession and/or tag is likely to change... but now that I write that out, I'm not so sure. Some of the datasets are just specialized versions of others, so they might have more overlap than I think.)